packages/kit/src/views/Discovery/pages/Dashboard/Welcome/Welcome.tsx

```
@@ -1,27 +1,104 @@
+
+import { useMemo } from 'react';
+
+import { Skeleton, Stack, XStack } from '@onekeyhq/components';
```

ESLint

[error] '@onekeyhq/components' imported multiple times.

(import/no-duplicates)